### PR TITLE
NAS-107062 / 20.10 / Nas 107062

### PR DESCRIPTION
**Pull request by lola27**

### CODE REVIEW

**Comment by lola27** [comment]
Screen Shot 2020-09-18 at 3 41 43 PM
Screen Shot 2020-09-18 at 3 40 10 PM
Screen Shot 2020-09-18 at 3 40 26 PM
Screen Shot 2020-09-18 at 3 30 48 PM
Screen Shot 2020-09-18 at 3 41 03 PM
Screen Shot 2020-09-18 at 3 39 29 PM

**Review by Mrt134 — COMMENTED** [review]
Quick look at changes seems fine - I'll add help text to this PR in just a little while.

**Review by william-gr — APPROVED** [review]
LGTM!

**Review by kmoore134 — APPROVED** [review]
Fantastic work @lola27! Merge it :)

File: src/app/helptext/network/configuration/configuration.ts [hunk 1]
```diff
@@ -83,4 +85,22 @@ wsd_placeholder: T('WS-Discovery'),
 wsd_tooltip: T('Uses the SMB Service <i>NetBIOS Name</i> to advertise \
  the server to WS-Discovery clients. This causes the computer appear in \
  the <i>Network Neighborhood</i> of modern Windows OSes.'),
+
+ outbound_network_activity: {
+    allow: {
+        placeholder: T('Allow All'),
+        tooltip: T(''),
```
Comment (Mrt134) [inline src/app/helptext/network/configuration/configuration.ts:92]:
```suggestion
        tooltip: T('Any system service can communicate externally.'),
```

File: src/app/helptext/network/configuration/configuration.ts [hunk 2]
```diff
@@ -83,4 +85,22 @@ wsd_placeholder: T('WS-Discovery'),
...
+    },
+    deny: {
+        placeholder: T('Deny All'),
+        tooltip: T(''),
```
Comment (Mrt134) [inline src/app/helptext/network/configuration/configuration.ts:96]:
```suggestion
        tooltip: T('This system cannot communicate externally.'),
```

File: src/app/helptext/network/configuration/configuration.ts [hunk 3]
```diff
@@ -83,4 +85,22 @@ wsd_placeholder: T('WS-Discovery'),
...
+    },
+    specific: {
+        placeholder: T('Allow Specific'),
+        tooltip: T(''),
```
Comment (Mrt134) [inline src/app/helptext/network/configuration/configuration.ts:100]:
```suggestion
        tooltip: T('Define the system services that are allowed to communicate externally. All other external traffic is restricted.'),
```

File: src/app/helptext/network/configuration/configuration.ts [hunk 4]
```diff
@@ -83,4 +85,22 @@ wsd_placeholder: T('WS-Discovery'),
...
+    },
+ },
+ outbound_network_value: {
+     tooltip: T(''),
```
Comment (Mrt134) [inline src/app/helptext/network/configuration/configuration.ts:104]:
```suggestion
     tooltip: T('Select the system services that will be allowed to communicate externally.'),
```

**Review by Mrt134 — APPROVED** [review]
Suggested help text for new fields.

**Review by erinix — APPROVED** [review]

**Pull request merged**